packages/admin/cms-admin/src/cronJobs/JobsGrid.tsx

```diff
@@ -35,6 +37,7 @@ const jobsQuery = gql`
         kubernetesJobs(cronJobName: $cronJobName) {
             id
             name
+            label
```

Comment:
```suggestion
```

I believe this isn't used

Reply:
fixed with 29fbb1440ce53933c5bf285fa0f56a99ea73e86d

initial idea was to avoid the additional query when navigating to logs, but this isn't working anyway